beancount.el

```diff
@@ -109,6 +109,13 @@ from the open directive for the relevant account."
   `((t :inherit highlight))
   "Face to highlight Beancount transaction at point.")
 
+(dotimes (i 8)
+  (let ((i (1+ i)))
+    (eval `(defface ,(intern (format "beancount-level-%d" i))
+             `((t :inherit ,(intern (format "outline-%d" i))))
+             ,(format "Level %d of `outline-minor-mode' headings in `beancount-mode'." i))
+          t)))
+
```

Comment:
This changes the faces used for fontification of the headlines. Anyhow, I would prefer to simply add a `(require 'org
)`

Reply:
I strongly hope you won't pull in all of `org` just for this small use case. Org's `org-level-N` faces inherit from these `outline-N` faces anyway, so whether it actually changes anything comes down to which faces your theme have styled.

In any case, you're styling outline headings, so using outline faces seems most appropriate.

Reply:
I think the discussion would be worth having if the impact of adding `(require 'org)` could be measurable in the time required to load beancount-mode, but it seriously doubt it is.

Reply:
An alternative: keep the `beancount-level-N` faces, have those inherit the org-level-N family, then themes can choose to style those directly. Users of these themes avoid the original issue and don't have to load org.

Reply:
If you inherit from `org` faces you need to load `org-mode` anyway, otherwise how is the face resolved?

Reply:
> If you inherit from `org` faces you need to load `org-mode` (or at least `org-faces`) anyway, otherwise how is the face resolved?

Yes, it doesn't fix the original issue, but it allows themes to override the faces directly (e.g. undoing the inheritance). Not to imply this is a perfect solution, just another one, if you're adamant on keeping the connection.